tsconfig.json

```diff
@@ -2,8 +2,10 @@
   "extends": "@tsconfig/node18",
   "compilerOptions": {
     "paths": {
-      "@final-ui/app": ["./packages/playground/src"],
+      "@final-ui/playground": ["./packages/playground/src"],
       "@final-ui/react": ["./packages/react/src"],
+      "@final-ui/react/server": ["./packages/react/src/server"],
+      "@final-ui/react/jsx-runtime": ["./packages/react/src/jsx"],
       "@final-ui/ws-client": ["./packages/ws-client/src"],
```

Comment:
In production, we use `lib`. In development, we use `src` and compile with TypeScript to save time (no need to run `watch` and re-compile `lib`). Added two more paths as per "exports" field and fixed incorrect package name.

Reply:
nice this sounds good